conf/test_full.config

```diff
@@ -10,6 +10,59 @@
 ----------------------------------------------------------------------------------------
 */
 
+process {
+withName: '.*:ASSEMBLE:.*:BUSCO' {
+        ext.prefix = { "${meta.id}_assembly-${lineage}" }
+        publishDir = [
+            path: { "${params.outdir}/${meta.id}/QC/BUSCO/assemble" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
```

Comment:
I think you don't need to override `ext.prefix` or the `publishDir` `mode` and `saveAs`, as they are the same in `modules.config`